apps/wallet-connect/src/hooks/useWalletConnectV2.tsx

```diff
@@ -168,8 +168,9 @@ const useWalletConnectV2 = (
       // events
       web3wallet.on('session_proposal', async proposal => {
         const { id, params } = proposal
-        const { requiredNamespaces } = params
+        const { requiredNamespaces, optionalNamespaces } = params
```

Comment:
We could remove this from this release as it's slightly out of scope.

But I realized that some dapps won't work without also taking the `optionalNamespaces` into account.